test/testcafe/spec/SessionStorage_spec.js

```diff
@@ -8,7 +8,10 @@ import ChallengeEmailPageObject from '../framework/page-objects/ChallengeEmailPa
 import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
 import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
 import TerminalPageObject from '../framework/page-objects/TerminalPageObject';
-import { getStateHandleFromSessionStorage } from '../framework/shared';
+
+export const getStateHandleFromSessionStorage = ClientFunction(() => {
```

Comment:
i guess this change was unintentional..

Reply:
It was intentional. This function is only used in this test class and contains the appId specific to these tests, which can be different from others.

Reply:
then why to remove `getStateHandleFromSessionStorage` from index.js, may be it can be used later somewhere? probably rename it to `getStateHandleAppIDFromSessionStorage`